src/praisonai-sandbox/praisonai_sandbox/daytona.py

```
@@ -1,106 +1,113 @@
-            self._workspace = None
-        
-        self._client = None
+        if not self._is_running:
+            return
+        sandbox = self._sandbox
+        self._sandbox = None
         self._is_running = False
+        if sandbox is not None:
+            try:
+                await asyncio.to_thread(sandbox.delete)
+            except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)